tasks/steps/upload_app_config.yml

```diff
@@ -0,0 +1,27 @@
+---
+
+- run_once: true
+  delegate_to: '{{ control_instance.name }}'
+  when: cartridge_app_config_path is not none
+  tags: cartridge-config
+  block:
+    - name: 'Copy application config on remote host'
+      any_errors_fatal: true
+      copy:
+        src: '{{ cartridge_app_config_path }}'
+        dest: '/tmp/'
+
+    - name: 'Apply application config'
+      cartridge_apply_app_config:
+        local_config_path: '{{ cartridge_app_config_path }}'
+        remote_dir: '/tmp/'
+        upload_endpoint: '{{ cartridge_app_config_upload_endpoint }}'
+        console_sock: '{{ control_instance.console_sock }}'
+        http_port: '{{ cartridge_app_config_upload_http_port or control_instance.http_port }}'
```

Comment:
Why do you allow to specify `cartridge_app_config_upload_http_port`? Because we can select control instance without HTTP enabled?

Reply:
Yes, you are right. I don't see any point in making a complex automatic system for selecting an HTTP port, since the HTTP port is almost always enabled